cns/saas/admin_guide/reference/syslog-forwarding.adoc

```diff
@@ -0,0 +1,96 @@
+= Syslog Forwarding
+Alexandre Cezar <acezar@paloaltonetworks.com> 1.0, April 18, 2022:
+
+== Enabling Syslog forwarding
+
+Syslog forwarding is enabled under the Enforcer Profile. Optionally, you can configure the header format used in syslog messages and enable client authentication for syslog over TLS.
+
```

Comment:
```suggestion

If you enable syslog forwarding without defining a syslog server endpoint, enforcers write events to the host subsystem. This can be useful if you already have a syslog collector in your host, but it only works for Linux based systems and on environments where your enforcers have permissions to write to the platform's disk. Many managed platforms don't allow your enforcers to write to the platform's disk which will cause an error.

```